docs/website/layouts/index.html

```diff
@@ -725,8 +744,8 @@ <h2>Declarative Policy</h2>
                               <div class="text-center blue">
                                 <button
                                     class="mt-4 btn btn-primary btn-playground"><a
-                                    target=”_blank”
-                                    href="https://play.openpolicyagent.org/p/5loFkqN19G">Open In Playground
```

Comment:
See? Two different links before the replacement. Forgotten to update in a previous PR. 🙃